roles/tarzan/meta/main.yml

```
@@ -1,3 +1,4 @@
 dependencies:
-  - { role: setup_base_config, tags: setup_base_config }
+  - { role: setup_base_config, tags: setup_base_config,
+      role: func_accounts, tags: func_accounts}
```

```suggestion
  - { role: setup_base_config, tags: setup_base_config }
  - { role: func_accounts, tags: func_accounts }
```
Newbie question here, shouldn't each role be a separate item in the list?